### PR TITLE
Look this PR

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**

### CODE REVIEW

File: VulnerableApplication/Controllers/HomeController.cs [hunk 1]
```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        public static string ChangePassword = "PassWord917231";
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Change the visibility of 'ChangePassword' or make it 'const' or 'readonly'.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Remove this method and declare a constant for this value.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Change the visibility of 'ChangePassword' or make it 'const' or 'readonly'.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Remove this method and declare a constant for this value.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Change the visibility of 'ChangePassword' or make it 'const' or 'readonly'.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Remove this method and declare a constant for this value.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem
